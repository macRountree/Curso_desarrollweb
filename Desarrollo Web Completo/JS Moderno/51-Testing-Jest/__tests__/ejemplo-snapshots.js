const cliente = {
  nombre: 'Mac',
  balance: 500,
  tipo: 'Premium',
};

describe('Test al cliente', () => {
  test('should Mac', () => {
    expect(cliente.nombre).toMatchSnapshot();
  });
});
