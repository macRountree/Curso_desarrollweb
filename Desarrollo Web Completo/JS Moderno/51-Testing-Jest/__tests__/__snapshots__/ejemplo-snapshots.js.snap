// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test al cliente should Mac 1`] = `"Mac"`;
